addons/sourcemod/scripting/include/KnifeAlert.inc

```diff
@@ -0,0 +1,55 @@
+#if defined _knifealert_included
+#endinput
+#endif
+#define _knifealert_included
+
+/**
+ * Called right after an Human knife a Zombie
+ *
+ * @param attacker       Attacker UserId
+ * @param victim         Victim UserId
+ * @param damage         Damage made by attacker to victim.
+ * @noreturn
+ */
+forward void KnifeAlert_OnKnife(int attacker, int victim, int damage);
+
+/**
+ * Called right after an Human knife a Zombie
+ *
+ * @param attacker          Attacker UserId
+ * @param Auth_attacker     Attacker SteamID  
+ * @param victim            Victim UserId
+ * @param pOldKnifer        Initial knifer UserId
+ * @param Auth_OldKnifer    Initial knifer SteamID 
+ * @noreturn
+ */
+forward void KnifeAlert_OnInfection(int attacker, char[] Auth_attacker, int victim, int pOldKnifer, char[] Auth_OldKnifer);
```

Comment:
we can use the zombiereloaded forwards no ?

Reply:
Zombie Reloaded is optional, plugin can be used for others game mod.